docs/topics/auth/passwords.txt

```diff
@@ -285,24 +285,31 @@ follows:
 
 scrypt_ has the following attributes that can be customized:
 
-#. ``work_factor`` controls the number of iterations within the hash.
-#. ``block_size``
-#. ``parallelism`` controls how many threads will run in parallel.
+#. ``work_factor`` controls the computational and memory cost parameter (*N*);
+   must be a power of 2.
+#. ``block_size`` controls the internal block size (*r*), tuning the algorithm
+   to memory latency.
+#. ``parallelism`` controls how many independent computations may run in
+   parallel (*p*).
```

Comment:
I think that mentioning `N`, `r`, and `p` is confusing, not helpful, especially that we use ``work_factor``, ``block_size``, and ``parallelism`` in the following explanation. I'd remove them.